nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -61,8 +61,7 @@ def update_ema(biased_ema, value, decay, step):
     float, float
     """
     biased_ema = biased_ema * decay + (1 - decay) * value
-    unbiased_ema = biased_ema / (1 - decay ** step)  # Bias correction
```

Comment:
seems that u cant remove unbiased ema here. [](https://img2018.cnblogs.com/blog/1351564/201906/1351564-20190615000526560-254363648.png)

Reply:
check this out [post](https://www.cnblogs.com/wuliytTaotao/p/9479958.html)

Reply:
![key picture here](https://user-images.githubusercontent.com/39682259/102965046-ffa55b00-4527-11eb-893c-1f081e0cf7da.png). Calculate Unbiased EMA seems more friendly to the beginning of training procedure?

Reply:
Good point. Have deleted `step`.
**Unbiased_ema** is used to solve the problem of deviation during ema initialization. But this method have bad performance when testing some network designed for mobile device such as mobilenetv2. In this case, mobilenetv2 can't converge at all. We add code below to resolve this problem:
`module.tracked_min_biased, module.tracked_max_biased = torch.min(output), torch.max(output)`
and we don't need **unbiased_ema** anymore. I have test the accuracy in mobilenetv2, lenet and shufflenetv2. All of them have better or same performance.

Reply:
Initialize track_min and track_max is good and this does have a good influence on faster && better converge, which is the key for your mobilenetV2 and shuffulenetV2 experiments because they are more alert to quantization. But I dont think we can draw a conclusion that unbiased_ema is not needed anymore?

Reply:
Or did u do any experiment that with the same initialization method, remove unbiased ema does help to converge faster and better?

Reply:
Yeah, you mention the key point. The difference of whether removing `unbiased_ema` reveals when `quant_start_step` is small. I have done the experiment setting `quant_start_step` to **5** on mobilenetv2. In this case, if we keep `unbiased_ema`, the model can't converge. And when we remove it, the model can converge normally.

Reply:
Um，i think I didn’t  make myself clear.
You made 2 changes here
1. Initialize trackmin and trackmax to min and max of the tensor, which is good.
2. You removed unbiased ema, which I am confused about,

From my view, your  experiments above proved your first modification, which I think is reasonable. THUMBS UP ！
And your experiment above didn’t give us any clue that unbiased ema hurts convergence. I think 4 experiments should be done
(1). With and Without modification 1. Which you have done I think. GOOD.
(2) With and Without unbiased ema but BOTH WITH MODIFICATION 1

Reply:
The experiment `quant_start_step` to **5** mentioned above is on the experiment with modification 1. In this case, if we keep unbiased_ema, the model can't converge. And when we remove it, the model can converge normally.  And I also test 'quant_start_step' **3**, **4**, the result is similar. 
If we set `quant_start_step` to large number like **1000**, there will be little difference between adding **unbiased_ema** and keeping **unbiased_ema**. 
Based on the experiments, delete **unbiased_ema** performs good during the situation when `quant_start_step` is small. And I choose to delete it.

Reply:
Oh yeah, that is what I mean.Good!